src/app/view/components/dot-add-persona-dialog/dot-add-persona-dialog.component.ts

```diff
@@ -0,0 +1,97 @@
+import { Component, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { DotMessageService } from '@services/dot-messages-service';
+import { take } from 'rxjs/operators';
+import { DotDialogActions } from '@components/dot-dialog/dot-dialog.component';
+import { DotCreatePersonaFormComponent } from '@components/dot-add-persona-dialog/dot-create-persona-form/dot-create-persona-form.component';
+import { FormGroup } from '@angular/forms';
+import { DotPersona } from '@models/dot-persona/dot-persona.model';
+import { DotWorkflowActionsFireService } from '@services/dot-workflow-actions-fire/dot-workflow-actions-fire.service';
+
+const PERSONA_CONTENT_TYPE = 'persona';
+
+@Component({
+    selector: 'dot-add-persona-dialog',
+    templateUrl: './dot-add-persona-dialog.component.html',
+    styleUrls: ['./dot-add-persona-dialog.component.scss']
+})
+export class DotAddPersonaDialogComponent implements OnInit {
+    @Input() visible = false;
+    @Output() persona: EventEmitter<DotPersona> = new EventEmitter();
```

Comment:
what is this output mean? I think that the name is not right because it does not tell anything